### PR TITLE
[NA] Adds arguments to unit tests

### PR DESCRIPTION
**Pull request by rypete**
Adds arguments for unit tests matching the capabilities of integration tests.

### CODE REVIEW

File: src/cli.ts [hunk 1]
```diff
@@ -78,26 +78,19 @@ program.command('docs')
...
-program.command('test:unit')
-    .description('Runs all unit tests for the extending project.')
-    .action(runUnitTests);
-
-/**
- * Integration tests command
- */
-program.command('test:int')
-    .description('Runs all integration tests for the extending project.')
+program.command('test <type>')
```
Comment (pjbroadbent) [inline src/cli.ts:83]: Can you document the `type` arg? Guessing allowed values are `"unit"` and `"int"`?

File: src/testing/runner.ts [hunk 2]
```diff
@@ -28,62 +27,32 @@ const fail = (err: string) => {
     process.exit(1);
 };
 
-const run = (...args: any[]) => {
-    const p = execa(...args);
+const run = (processName: string, args?: any[], execaOptions?: any) => {
```
Comment (pjbroadbent) [inline src/testing/runner.ts]: Does `execa` include type definitions for it's options?

File: src/cli.ts (hunk 1 above)
Comment (rypete) [reply src/cli.ts:83]: Is now better documented in the description

File: src/testing/runner.ts (hunk 2 above)
Comment (rypete) [reply src/testing/runner.ts]: Yes it does.  I was going to do this yesterday but it was causing issues where we used it.  Realised I was being silly so fixed now

**Review by pjbroadbent — APPROVED** [review]

**Pull request merged**